smoke-tests/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ dependencies {
   testImplementation("io.opentelemetry:opentelemetry-proto")
   testImplementation("io.opentelemetry:opentelemetry-api")
 
-  testImplementation("ch.qos.logback:logback-classic:1.2.3")
+  testImplementation("ch.qos.logback:logback-classic:1.2.4-groovyless")
```

Comment:
what is this? I can't even see it in maven central

Reply:
ah there it is. still no idea what this might be, though: https://search.maven.org/artifact/ch.qos.logback/logback-classic/1.2.4-groovyless/jar

Reply:
found this in their jira: https://jira.qos.ch/browse/LOGBACK-1564?jql=text%20~%20%22groovyless%22

Reply:
I think I even support this upgrade :)

Reply:
It sounds good for sure, but is this an official release from the project, or just something someone pushed ad-hoc? I'm a little nervous about picking up ad-hoc library versions that might not be in the mainline code.

Reply:
Ceki seems to be the author of logback